data/yaml/page-content/livesync.yaml

```diff
@@ -23,17 +25,24 @@ sections:
     columns: 2
     bottomMargin: 96
     cards:
-      - title: Frontend data models
+      - title: Postgres Database Connector
         type: feature
-        content: The Models SDK sits in your frontend apps to simplify subscribing to changes in data models.
-        image: 'models-sdk.svg'
+        content: The Postgres connector connects to your postgres database outbox table to monitor and broadcast changes at scale in realtime.
+        image: 'database-connector.svg'
         links:
           - text: 'Read docs'
-            href: '/livesync/models'
-      - title: Database Connector
+            href: '/livesync/postgres-database-connector'
+      - title: MongoDB Database Connector
         type: feature
-        content: The Database Connector connects to your database to monitor and broadcast changes at scale in realtime.
+        content: The MongoDB connector connects to your database to broadcast document change events from the ChangeStreamAPI at scale in realtime.
         image: 'database-connector.svg'
         links:
           - text: 'Read docs'
-            href: '/livesync/database-connector'
+            href: '/livesync/postgres-database-connector'
+      - title: Frontend data models
```

Comment:
I think this card should be positioned next to the postgres connector one

Reply:
Before:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/1c439417-c943-4cac-abbf-be829acfadbf">


After:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/caf78c1f-6d02-4747-b998-b4e4adf95da9">

Reply:
I'd suggest we just have 'Postgres' and 'MongoDB'. I think the models side will complicate things since Mongo doesn't use that.